lib/core/models/responses/all_spaces_response.model.dart

```diff
@@ -17,18 +17,39 @@
   along with this program.  If not, see <https://www.gnu.org/licenses/>.
 */
 
-import '../space.model.dart';
-import '../user.model.dart';
-import '../member.model.dart';
+import 'package:demos_app/core/models/manifesto/comment/manifesto_comment.model.dart';
+import 'package:demos_app/core/models/manifesto/comment/manifesto_comment_vote.model.dart';
+import 'package:demos_app/core/models/manifesto/manifesto.model.dart';
+import 'package:demos_app/core/models/manifesto/manifesto_option.model.dart';
+import 'package:demos_app/core/models/manifesto/proposal/proposal.model.dart';
+import 'package:demos_app/core/models/manifesto/proposal/proposal_participation.model.dart';
+import 'package:demos_app/core/models/manifesto/proposal/proposal_vote.model.dart';
+import 'package:demos_app/core/models/member.model.dart';
+import 'package:demos_app/core/models/space.model.dart';
+import 'package:demos_app/core/models/user.model.dart';
 
 class AllSpacesResponse {
   late List<Space> _spaces;
   late List<Member> _members;
   late List<User> _users;
+  late List<Proposal> _proposals;
```

Comment:
Cambia el late por el final, y crea un constructor que inicialize todos los atributos y mándalo a llamar desde el factory fromObject, como lo hacemos en todos los modelos 

class AllSpacesResponse {
  final List<Space> _spaces;
  ....
   AllSpacesResponse(this._spaces, ....):

  factory AllSpacesResponse.fromObject(dynamic o) {
   List<Space> _spaces = ....

   return AllSpacesResponse(_spaces, ....);
}